### DISCUSSION TITLE
802

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#977] [Comments]
# 802

늦은 시간에 퇴근을 하면 종종 신기한 경험을 하게 된다.너무 늦은 시간에 퇴근을 하면 보통은 바로 회사 택시를 불러 타고 가곤 하는데, 그날은 머리도 식힐겸 조금은 걷다가 택시를 타려고 했다.저녁 11시가 넘는 늦은 시간.주변에 아무도 없는, 지나가는 차도 없는 아주 조용한 횡단보도의 신호등을 기다리는 사람을 본 적이 있다.지켜보는 사람 하나 없는 곳에서도 규칙을 지키는 사람이 있다는 것에 걸음을 멈추고 한동안 지켜보게 되었다.신호등의 불이 켜지고나서야 건너가는 모습을 보면서 멋지다는 생각을 했다.누군가에게는 융통성이 없는 것일 수 있다.지나가는 차 하나 없어 사고날 일이 없고,지켜보는 사람도 없으니 누군가 이상한 눈길로 볼 일도 없을 것이다.그러면 굳이 기다리지 않고 지나가는게 늦은 시간 빨리 집에 갈..

https://jojoldu.tistory.com/802?utm_campaign=asb&utm_medium=blog&utm_source=awesome-blogs.petabytes.org

**Comment by bellfive** [comment]
정말 좋은 글이라 댓글 남깁니다~

우리 아버지가 해주신 말씀이 기억나네요. 아버진 경찰이셨는데...

"사람들이 경찰이나 택시 기사들이 정말로 문제가 많다. 뒤로 해먹거나 운전을 잘 못한다라고 욕하곤 하는데..
우리 나라가 이렇게 잘 돌아가는 것은 뉴스에 나온 문제가 있는 사람보다는 뒤에서 묵묵히 일하는 사람들이 많다는 걸 뜻한다."

뉴스만 보고 한직군을 욕하지 말란 뜻도 같이 포함해서 말씀하시긴 한 것이지만, 남이 보지 않아도 묵묵히 자기 룰을 지키는 사람과 일맥상통한다고 느껴져서 첨언합니다.

좋은 글 잘 봤습니다 ^^
